Gopkg.toml

```diff
@@ -22,6 +22,11 @@ required = [
   name = "github.com/percona-platform/saas"
   branch = "main"
 
+# FIXME Remove when https://jira.percona.com/browse/PMM-6629 is done in percona/pmm.
```

Comment:
What is the reason to update protobuf in `managed` without updating it in `pmm`?

Reply:
https://github.com/percona/pmm-managed/pull/451#issuecomment-692748135